src/main/java/io/kontur/disasterninja/controller/AnalyticsTabController.java

```diff
@@ -11,15 +13,13 @@
 import lombok.RequiredArgsConstructor;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import org.wololo.geojson.GeoJSON;
 
 import java.util.List;
 
 @Tag(name = "Analytics tab", description = "Analytics tab API")
 @RestController
-@RequestMapping("/polygon_details")
```

Comment:
Sorry, I think I misunderstood you during our sync. Maybe it's better to leave this path for the whole controller and rather use `/v2` in a new method so it has full path `/polygon_details/v2`? Since versioning is related to `polygon_details` functionality only and not the whole API